tests/measure.py

```diff
@@ -99,7 +100,9 @@ def __eq__(self, other):
 
     def setUp(self):
         ms = afwImage.MaskedImageF(afwGeom.ExtentI(31, 27))
-        var = ms.getVariance(); var.set(1); del var
+        var = ms.getVariance()
+        var.set(1)
+        del var
```

Comment:
Sure. It's out of scope, but too trivial to justify another ticket so I'll implement your suggested modernizations in a separate commit.